token-syncer/src/token_syncer/main.clj

```diff
@@ -37,7 +38,12 @@
 (defn parse-cli-options
   "Parses and returns the cli options passed to the token syncer."
   [args]
-  (->> [["-d" "--dry-run"
+  (->> [["-a" "--backup-accrete"
+         "Accrete new token descriptions into the backup file if it exists instead of overwriting (default behavior) the file."]
+        ["-b" "--backup-file file"
```

Comment:
Does the CLI library support subcommands?  Might be nice to have separate `sync` and `backup` commands, with `--accrete` (if necessary) an option only supported for `backup`.

Reply:
What does it mean to have a `--dry-run` of a backup operation?  Perhaps it's only useful for `sync`.

Reply:
Should we take this opportunity to rename this utility from `token-syncer` to `waiter-admin`